caas/kubernetes/provider/bootstrap.go

```diff
@@ -505,8 +507,12 @@ func (c controllerStack) ensureControllerConfigmapAgentConf() error {
 	return c.broker.ensureConfigMap(cm)
 }
 
-func (c controllerStack) createControllerStatefulset() error {
-	numberOfPods := int32(1) // TODO: HA mode!
+func getPodName(pcfg *podcfg.ControllerPodConfig) string {
```

Comment:
This package function should be either called getControllerPodName() or preferably live on the ControllerPodConfig receiver. It's too generic at the moment.

Reply:
moved to podcfg package